eclipse-theia/theia
# fix(commad): close #11346, use available number to label when create …

#### What it does


close #11346

after change:

![QQ20220626-180740](https://user-images.githubusercontent.com/11473889/175809833-cdcb6ec6-b5b2-494c-a6f0-f965704ad6f6.gif)

#### How to test


1. yarn & yarn browser build & yarn browser start.
2. Open `http://localhost:3000/` and open any workspace.
3. Use `alt + n` to call `NEW_FILE` command some times, we can see tab named `Untitled-${index}` one by one. The number in label is start from `1` like vscode.
4. Call `NEW_FILE` command again after close some older Untitled file, it will use the number which is closed just now like vscode.

#### Review checklist


#### Reminder for reviewers

- As a reviewer, I agree to behave in accordance with [the review guidelines](https://github.com/theia-ide/theia/blob/master/doc/pull-requests.md#reviewing)
